src/main/java/model/process/SortProcess.java

```diff
@@ -0,0 +1,67 @@
+package model.process;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import model.data.DataRow;
+import model.data.DataTable;
+import model.data.DataTableConversionBuilder;
+import model.data.Table;
+import model.data.describer.RowValueDescriber;
+import model.data.value.DataValue;
+
+/**
+ * This class sorts a DataTable on whatever column you want it to sort on.
+ * 
+ * @author Louis Gosschalk 09-06-2015
+ */
+public class SortProcess extends DataProcess {
+	private DataTable table;
+	private RowValueDescriber<DataValue> column;
+	private Order order;
+
+	public SortProcess(DataTable table, RowValueDescriber<DataValue> column, Order order) {
```

Comment:
Nee, ik krijg een nullpointer wanneer ik process uitvoer na de table als input ingesteld te hebben

Is inderdaad een goed punt, ik zal zorgen dat het via getInput werkt